drivers/gpio/gpio_nrf5.c

```diff
@@ -386,6 +449,18 @@ static const struct gpio_driver_api gpio_nrf5_drv_api_funcs = {
 	.disable_callback = gpio_nrf5_disable_callback,
 };
 
+static u32_t gpiote_chan_mask;
```

Comment:
I believe this was defined above at line 69.

Reply:
good catch, will fix it. Would why i got not compiler error/warning?! map file correctly had a single instance allocated.